src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1754,36 +1755,26 @@ private DetectProperties() {
-                    "Detect logging uses Spring Boot logging, which uses Logback (https://logback.qos.ch). " +
-                    "The format of this property name is <i>logging.level.{package}[.{class}]</i>. " +
-                    "The property name shown above specifies package <i>com.synopsys.integration</i> because that is the name of Detect's top-level package. " +
-                    "Changing the logging level for that package changes the logging level for all Detect code, as well as Synopsys integration libraries that Detect uses. " +
-                    "Non-Synopsys libraries that Detect uses are not affected. " +
-                    "However, you can use this property to set the logging level for some of the non-Synopsys libraries that Detect uses by using the appropriate package name. " +
-                    "For example, <i>logging.level.org.apache.http=TRACE</i> sets the logging level to TRACE for the Apache HTTP client library. " +
-                    "<p/>" +
-                    "For log message format, a default value of <i>%d{yyyy-MM-dd HH:mm:ss z} ${LOG_LEVEL_PATTERN:%-6p}[%thread] %clr(---){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}</i> is used. "
-                    +
-                    "You can change your log message format by setting the Spring Boot <i>logging.pattern.console</i> property to a different pattern. " +
-                    "<p/>" +
-                    "Refer to the Spring Boot logging and Logback Project documentation for more details."
-            )
-            .setGroups(DetectGroup.LOGGING, DetectGroup.GLOBAL)
-            .build();
-
     public static final EnumProperty<LogLevel> LOGGING_LEVEL_DETECT =
         EnumProperty.newBuilder("logging.level.detect", LogLevel.INFO, LogLevel.class)
-            .setInfo("Logging Level Shorthand", DetectPropertyFromVersion.VERSION_5_5_0)
-            .setHelp(
-                "Shorthand for the logging level of detect. Equivalent to setting <i>logging.level.com.synopsys.integration</i>.",
-                "Refer to the description of property <i>logging.level.com.synopsys.integration</i> for additional details."
-            )
+            .setInfo("Logging Level", DetectPropertyFromVersion.VERSION_5_5_0)
+            .setHelp(
+                    "The logging level of Detect.",
+                    "To keep the log file size manageable, use INFO level logging for normal use and DEBUG or TRACE for troubleshooting.<p/>" +
+                        "Detect logging uses Spring Boot logging, which uses Logback (https://logback.qos.ch). " +
+                        "The format of this property name is <i>logging.level.{package}[.{class}]</i>. " +
+                        "The property name shown above specifies package <i>com.synopsys.integration</i> because that is the name of Detect's top-level package. " +
```

Comment:
Is this line still accurate now that the property name has changed?
I also wonder if other usages of the word "Synopsys" should be changed in this help message.